src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -37,6 +37,8 @@ public sealed partial class Window
 				+= RaiseNativeSizeChanged;
 		}
 
+		internal Activity NativeWindow => ApplicationActivity.Instance;
```

Comment:
So there's nothing higher than an Activity for Android?

Reply:
Good point! There is one - and I adjusted the code to return `Android.Views.Window` instead, which still can access the activity if needed 👍